cirq-core/cirq/protocols/json_test_data/GridDeviceMetadata.json

```diff
@@ -155,5 +155,17 @@
         "picos": 1
       }
     ]
+  ],
+  "isolated_qubits": [
```

Comment:
include a `json_inward` file using the original json data file (unmodified) to make sure you can still load old versions

Reply:
See above.